spanner/ddl/ast.go

```diff
@@ -191,8 +192,9 @@ func (cd ColumnDef) PrintColumnDef(c Config) (string, string) {
 //     key_part:
 //        column_name [{ ASC | DESC }]
 type IndexKey struct {
-	Col  string
-	Desc bool // Default order is ascending i.e. Desc = false.
+	Col   string
+	Desc  bool // Default order is ascending i.e. Desc = false.
+	Order int
```

Comment:
We should store the order of primary key for source schema as well in session file